Gemfile.lock

```diff
@@ -284,7 +284,7 @@ GEM
     pry-rails (0.3.9)
       pry (>= 0.10.4)
     psych (3.1.0)
-    public_suffix (4.0.3)
+    public_suffix (4.0.4)
```

Comment:
an appropriate version number for a domain related gem 😆

Reply:
haha